### ISSUE TITLE
About the Pickscore performance

### ISSUE THREAD

**Issue by happynear** [NVlabs/DiffusionNFT#35]
Hi, recently I want to replicate the PickScore experiment. Now the eval reward is less than 0.9, which is about 23.4 PickScore. It's now 600 iterations. I don't think it could reach 23.8(0.915) in the paper if I continue the training. Could you provide the hyperparameters to replicate the performance reported in the paper?

<img width="555" height="484" alt="Image" src="https://github.com/user-attachments/assets/10885cfd-19c1-4668-9fa9-03afb1f8627c" />

**Comment by Henglin-Liu** [comment]
> Hi, recently I want to replicate the PickScore experiment. Now the eval reward is less than 0.9, which is about 23.4 PickScore. It's now 600 iterations. I don't think it could reach 23.8(0.915) in the paper if I continue the training. Could you provide the hyperparameters to replicate the performance reported in the paper?
> 
> <img alt="Image" width="555" height="484" src="https://private-user-images.githubusercontent.com/3897999/533132430-10885cfd-19c1-4668-9fa9-03afb1f8627c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzQ4NDA1NTksIm5iZiI6MTc3NDg0MDI1OSwicGF0aCI6Ii8zODk3OTk5LzUzMzEzMjQzMC0xMDg4NWNmZC0xOWMxLTQ2NjgtOWZhOS0wM2FmYjFmODYyN2MucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI2MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNjAzMzBUMDMxMDU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2VhMDQxMmQ5YTdmNjk1MWIzNmVjNTUyZmNjNWI5NjljM2QyZDcwMzZlYTJhODA0NzUzYmVkOTRkNWU3YzgxNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.19EO5owKn3uFg5vH-uREv8uSm5JX48rtle8xkfSEbtY">

Are you using the [configuration](https://github.com/NVlabs/DiffusionNFT/blob/cbb14f84b8312b620390dfcbe2fab69c1383b104/config/nft.py#L97) directly? With 8 GPUs, the reward I'm getting is rising very slowly.

<img width="1315" height="239" alt="Image" src="https://github.com/user-attachments/assets/270eff04-55e3-42b6-a849-4284fe825d63" />
